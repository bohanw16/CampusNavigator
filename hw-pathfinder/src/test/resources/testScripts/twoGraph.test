# test loading two graphs

CreateGraph g1

AddNode g1 n1
AddNode g1 n2
AddNode g1 n3

AddEdge g1 n1 n2 1.111
AddEdge g1 n2 n3 1.111

FindPath g1 n1 n3

CreateGraph g2

AddNode g2 n1
AddNode g2 n2
AddNode g2 n3

AddEdge g2 n1 n2 1.111
AddEdge g2 n2 n3 1.111

FindPath g2 n1 n3