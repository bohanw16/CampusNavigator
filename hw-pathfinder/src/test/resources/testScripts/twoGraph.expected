# test loading two graphs

created graph g1

added node n1 to g1
added node n2 to g1
added node n3 to g1

added edge 1.111 from n1 to n2 in g1
added edge 1.111 from n2 to n3 in g1

path from n1 to n3:
n1 to n2 with weight 1.111
n2 to n3 with weight 1.111
total cost: 2.222

created graph g2

added node n1 to g2
added node n2 to g2
added node n3 to g2

added edge 1.111 from n1 to n2 in g2
added edge 1.111 from n2 to n3 in g2

path from n1 to n3:
n1 to n2 with weight 1.111
n2 to n3 with weight 1.111
total cost: 2.222